### PR TITLE
Add an example for modbus encoding

### PR DESCRIPTION
**Pull request by relu91**
A minor PR with an example of how bytes are encoded using different byte orders.

### CODE REVIEW

**Review by danielpeintner — APPROVED** [review]
LGTM

test failures are unrelated

File: packages/binding-modbus/README.md [hunk 1]
```diff
@@ -93,6 +93,12 @@ Along with content type `application/octet-stream`, this protocol binding accept
 
 **Note**: the list may be extended in the future. 
 
+In particular, the decimal number `9545 22880` will be encoded in as follows:
```
Comment (danielpeintner) [inline packages/binding-modbus/README.md:96]: - decimal format seems a bit strange...  4 digits followed by 5 digits ?
- I am not native speaker but isn't it "will be encoded in as follows" --> "will be encoded as follows"

**Review by danielpeintner — COMMENTED** [review]
is the example correct?

decimal number 954522880 in octal 7071156400 ?
As mentioned I might not have enough insights how the mapping is done...

File: packages/binding-modbus/README.md (hunk 1 above)
Comment (relu91) [reply packages/binding-modbus/README.md:96]: yes sorry, my bad they are actually *two* decimal numbers. Fixing.

**Review by danielpeintner — APPROVED** [review]

**Pull request merged**